### PR TITLE
Benchmark prototype.

### PR DESCRIPTION
**Pull request by zhengrenjie**

### CODE REVIEW

File: benchmark/src/main/java/com/distkv/drpc/benchmark/common/common_pb.proto [hunk 1]
```diff
@@ -0,0 +1,14 @@
+syntax="proto3";
```
Comment (jovany-wang) [inline benchmark/src/main/java/com/distkv/drpc/benchmark/common/common_pb.proto:1]: Do not define this file. Just put the context to `string_pb.proto` and rename `string_pb.proto` to `str.proto`

File: benchmark/src/main/java/com/distkv/drpc/benchmark/common/consumer/DubboHttpProcessHandler.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
+package com.distkv.drpc.benchmark.common.consumer;
```
Comment (jovany-wang) [inline benchmark/src/main/java/com/distkv/drpc/benchmark/common/consumer/DubboHttpProcessHandler.java:1]: What's the purpose to use dubbo http process?

Does it mean we use dubbo rpc via HTTP?

**Review by jovany-wang — COMMENTED** [review]
Thanks. LGTM without the minor comments.

**Review by jovany-wang — APPROVED** [review]

**Review by jovany-wang — CHANGES_REQUESTED** [review]
Put benchmark into test module as a package

**Comment by zhengrenjie** [comment]
Closed.

**Pull request closed without merging**